src/librustc_incremental/calculate_svh/hasher.rs

```diff
@@ -50,23 +73,28 @@ impl Hasher for IchHasher {
     }
 
     #[inline]
+    fn write_u8(&mut self, i: u8) {
+        self.write_uleb128(i as u64);
```

Comment:
`u8` don't need to doesn't need to use leb128. There's nothing to be gained here.